pom.xml

```diff
@@ -350,15 +350,6 @@
                 <skip>true</skip>
               </configuration>
             </plugin>
-            <plugin>
-              <artifactId>maven-surefire-plugin</artifactId>
-              <configuration>
-                <systemPropertyVariables>
-                  <jacoco-agent.destfile>${jacoco.exec.file}</jacoco-agent.destfile>
-                </systemPropertyVariables>
-                <skip>${code.coverage.disabled}</skip>
```

Comment:
I was wondering about this property and grepped for it [1]. Does it make sense to define its value in all modules?
```
drools-wb-extensions/pom.xml:    <code.coverage.disabled>true</code.coverage.disabled>
pom.xml:    <code.coverage.disabled>false</code.coverage.disabled>
pom.xml:                <skip>${code.coverage.disabled}</skip>
pom.xml:              <skip>${code.coverage.disabled}</skip>
drools-wb-screens/drools-wb-guided-template-editor/pom.xml:    <code.coverage.disabled>true</code.coverage.disabled>
drools-wb-screens/drools-wb-factmodel-editor/pom.xml:    <code.coverage.disabled>true</code.coverage.disabled>
drools-wb-screens/drools-wb-drl-text-editor/pom.xml:    <code.coverage.disabled>true</code.coverage.disabled>
drools-wb-screens/drools-wb-guided-dtree-editor/pom.xml:    <code.coverage.disabled>true</code.coverage.disabled>
drools-wb-screens/drools-wb-test-scenario-editor/pom.xml:    <code.coverage.disabled>true</code.coverage.disabled>
drools-wb-screens/drools-wb-enum-editor/pom.xml:    <code.coverage.disabled>true</code.coverage.disabled>
drools-wb-screens/drools-wb-scenario-simulation-editor/pom.xml:    <code.coverage.disabled>false</code.coverage.disabled>
drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-api/pom.xml:    <code.coverage.disabled>true</code.coverage.disabled>
drools-wb-screens/drools-wb-globals-editor/pom.xml:    <code.coverage.disabled>true</code.coverage.disabled>
drools-wb-screens/drools-wb-scorecard-xls-editor/pom.xml:    <code.coverage.disabled>true</code.coverage.disabled>
drools-wb-screens/drools-wb-guided-dtable-editor/pom.xml:    <code.coverage.disabled>true</code.coverage.disabled>
drools-wb-screens/drools-wb-dtable-xls-editor/pom.xml:    <code.coverage.disabled>true</code.coverage.disabled>
drools-wb-screens/drools-wb-guided-rule-editor/pom.xml:    <code.coverage.disabled>true</code.coverage.disabled>
drools-wb-screens/drools-wb-guided-scorecard-editor/pom.xml:    <code.coverage.disabled>true</code.coverage.disabled>
drools-wb-screens/drools-wb-pmml-text-editor/pom.xml:    <code.coverage.disabled>true</code.coverage.disabled>
drools-wb-screens/drools-wb-workitems-editor/pom.xml:    <code.coverage.disabled>true</code.coverage.disabled>
drools-wb-screens/drools-wb-dsl-text-editor/pom.xml:    <code.coverage.disabled>true</code.coverage.disabled>
drools-wb-services/pom.xml:    <code.coverage.disabled>true</code.coverage.disabled>
```

Reply:
@jomarko 
If I remember correctly when test coverage metrics has been implemented in `drools-wb` most of the modules where not compatible with jacoco agent so we decided to disable it for all legacy editors with the idea to enable it back if and when we will have time to make coverage work. I don't know if this will happen (I think nobody is really working on legacy editors) so we can also decide to change default value to `true` and remove most of overrides.

Reply:
@danielezonca thank you let me clarify my understanding. So now when user open PR for drools-wb, code coverage will be measured just for the `drools-wb-scenario-simulation-editor` modules? Except `drools-wb-scenario-simulation-editor-api`?

Reply:
AFAIK yes (original [PR](https://github.com/kiegroup/drools-wb/pull/1217))

Reply:
Ok, I am interested in guided decision tables and rules https://issues.redhat.com/browse/BAQE-1225, approving this